release/tag/goreleaser.yaml

```diff
@@ -51,7 +51,7 @@ builds:
     env:
       - CC=x86_64-linux-gnu-gcc
       - CXX=x86_64-linux-gnu-g++
-    ldflags: -s -w -X github.com/GoogleContainerTools/kpt/run.version={{.Version}}
+    ldflags: -s -w -X github.com/GoogleContainerTools/kpt/run.version={{.Version}} -linkmode external -extldflags "-static"
```

Comment:
Looks like I accidentally left out the ldflags for linux arm64. Do you mind also adding the ldflags for the linux arm64 targets for uniformity? Thanks!

Reply:
Ah! Good catch. Done.